native-build.sh

```diff
@@ -13,4 +13,8 @@ curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain 1.75.0
 export PATH=$PATH:$CARGO_HOME/bin
 git config --global --add safe.directory /home/ubuntu/secp256r1/besu-native-ec
 git config --global --add safe.directory /home/ubuntu/secp256r1/besu-native-ec/openssl
+wget https://download.java.net/java/GA/jdk21.0.2/f2283984656d49d69e91c558476027ac/13/GPL/openjdk-21.0.2_linux-aarch64_bin.tar.gz
+tar -xzf openjdk-21.0.2_linux-aarch64_bin.tar.gz -C $HOME
+export JAVA_HOME=$HOME/jdk-21.0.2
+export PATH=$JAVA_HOME/bin:$PATH
```

Comment:
lets install openjdk-21-jdk from apt instead.  The only place this script is called from is a docker exec command in build.yml.  We should be able to run against 22.04 instead of 20.04 and will have openjdk-21-jdk available without having to do downloads of specific minor revisions

also, we should get rid of openjdk-11 in the apt install

Reply:
Removed openjdk-11 and added openjdk-21